### PR TITLE
[Android] Release notes for 1.10.x Hotfix 1

### LINKED ISSUE #10409: [Android] Release Notes for Android 1.10.x Hotfix 1 Release
* [Android] Disable twitter support in the panel #10208
* [Android] Ads on YT #10241
* [Android] Bottom toolbar setting reverted (possibly on update) #9843
* [Android] App crashes on opening new tab after disabling Custom Homepage theme #10296

### PR DESCRIPTION
**Pull request by kjozwiak**
Fix #10409

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] If adding a new directory with NPM packages, ensure this directory is
  added to `scripts/audit.js`.
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Test Plan:

N/A

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: CHANGELOG_ANDROID.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.10.95](https://github.com/brave/brave-browser/releases/tag/v1.10.95)
+
+- Disabled Twitter support in the rewards panel. ([#10208](https://github.com/brave/brave-browser/issues/10208))
+- Fixed "Enable bottom toolbar" being reverted in certain cases. ([#9843](https://github.com/brave/brave-browser/issues/9843))
+- Fixed ads being displayed under YouTube in certain cases. ([#10241](https://github.com/brave/brave-browser/issues/10241))
+- Fixed crash when opening new tab after disabling "Custom Homepage" theme. ([#10296](https://github.com/brave/brave-browser/issues/10296))
```
Comment (SergeyZhukovsky) [inline CHANGELOG_ANDROID.md:8]: I would add here on some screen resolutions

Comment (bsclifton) [reply CHANGELOG_ANDROID.md:8]: do you mean like `Fixed crash on some screen resolutions when opening new tab after disabling "Custom Homepage" theme`?

File: CHANGELOG_ANDROID.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.10.95](https://github.com/brave/brave-browser/releases/tag/v1.10.95)
+
+- Disabled Twitter support in the rewards panel. ([#10208](https://github.com/brave/brave-browser/issues/10208))
+- Fixed "Enable bottom toolbar" being reverted in certain cases. ([#9843](https://github.com/brave/brave-browser/issues/9843))
+- Fixed ads being displayed under YouTube in certain cases. ([#10241](https://github.com/brave/brave-browser/issues/10241))
```
Comment (srirambv) [inline CHANGELOG_ANDROID.md:7]: How about having it as "Fixed ads being shown on YouTube videos in certain cases"?

Comment (kjozwiak) [reply CHANGELOG_ANDROID.md]: I don't think we need "video" as it's basically implied that it's video content when mentioning YT. But used the above wording other than "video".

**Pull request merged**